files/scripts/syncd.sh

```diff
@@ -13,25 +9,25 @@ function debug()
 
 function lock_service_state_change()
 {
-    debug "Locking ${LOCKFILE} from ${SERVICE} service"
+    debug "Locking ${LOCKFILE} from ${SERVICE}$DEV service"
 
     exec {LOCKFD}>${LOCKFILE}
     /usr/bin/flock -x ${LOCKFD}
     trap "/usr/bin/flock -u ${LOCKFD}" 0 2 3 15
 
-    debug "Locked ${LOCKFILE} (${LOCKFD}) from ${SERVICE} service"
+    debug "Locked ${LOCKFILE} (${LOCKFD}) from ${SERVICE}$DEV service"
 }
 
 function unlock_service_state_change()
 {
-    debug "Unlocking ${LOCKFILE} (${LOCKFD}) from ${SERVICE} service"
+    debug "Unlocking ${LOCKFILE} (${LOCKFD}) from ${SERVICE}$DEV service"
     /usr/bin/flock -u ${LOCKFD}
 }
 
 function check_warm_boot()
 {
-    SYSTEM_WARM_START=`sonic-db-cli STATE_DB hget "WARM_RESTART_ENABLE_TABLE|system" enable`
-    SERVICE_WARM_START=`sonic-db-cli STATE_DB hget "WARM_RESTART_ENABLE_TABLE|${SERVICE}" enable`
+    SYSTEM_WARM_START=`/usr/bin/sonic-db-cli $DEV STATE_DB hget "WARM_RESTART_ENABLE_TABLE|system" enable`
```

Comment:
>/usr/bin/ [](start = 23, length = 9)

Can you keep original binary name only, without absolute path?

Reply:
Fixed.